src/components/MapControls/MapControls.js

```diff
@@ -121,11 +121,21 @@ function MapControls({onChange, mapDataset}) {
                 }
                 
                 //use indicator-specific colouration if available
-                const palette = (indicatorConfig && indicator in indicatorConfig) ?
-                    indicatorConfig[indicator].palette : 'x-Occam';
-                const logscale = (indicatorConfig && indicator in indicatorConfig) ?
-                    indicatorConfig[indicator].logscale : false;
-                
+                const config = indicatorConfig && indicator in indicatorConfig ?
```

Comment:
I think this can be simplified to `indicatorConfig?.[indicator]`. See [optional chaining operator.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)